README.md

```diff
@@ -1,2 +1,8 @@
-# opentelemetry service
-OpenTelemetry service
+# OpenTelemetry Service
+The OpenTelemetry service is a set of components that can collect traces, metrics and eventually other telemetry data (e.g. logs) from processes instrumented by OpenTelementry or other monitoring/tracing libraries (Jaeger, Prometheus, etc.), do aggregation and smart sampling, and export traces and metrics to one or more monitoring/tracing backends.
```

Comment:
also it does "enrichment" and in future transformation of telemetry. Like scrabbing of personal information or so...

Reply:
Sounds good, let me add that.